FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2DrawerActivity.kt

```diff
@@ -383,6 +383,8 @@ private fun getDrawerContent(
         lateinit var context: Context
         AndroidView(
             modifier = Modifier
+
+
```

Comment:
lets rem,ove this file from PR if no change

Reply:
removed